sidebars.json

```diff
@@ -0,0 +1,141 @@
+{
+    "docs": [
+        { "type": "link", "label": "Introduction", "href": "/" },
+        "getting-started",
+        {
+            "type": "category",
+            "label": "Scripting",
+            "items": [
+                "scripting/introduction",
+                {
+                    "type": "category",
+                    "label": "Abilities, items, modifiers",
+                    "items": [
+                        "scripting/abilities/datadriven-ability-breakdown-documentation",
```

Comment:
What happens when we add a new article? Do we have to manually add it here? Can we automate this?

Reply:
Yes, it has to be added there manually. It's possible use .js sidebars file and I have considered doing that, however if we'll generate the list we won't be able to sort them in the order users expect it to see

Reply:
I don't really like the DX for users that just want to add a new article, they have to edit two places now, as opposed to just creating a single new file in the current setup. But if there is no way around it I guess we can just pay extra attention to it in the PR.